pkg/printers/internalversion/describe.go

```diff
@@ -3829,33 +3829,26 @@ func printTolerationsMultilineWithIndent(w PrefixWriter, initialIndent, title, i
 	}
 
 	// to print tolerations in the sorted order
-	keys := make([]string, 0, len(tolerations))
```

Comment:
Why not using `sets.String` which for one will ensure you have exactly one key and then iterate over its `List` which returns sorted keys?

Reply:
`tolerations` is a list, where we cannot guarantee the key is unique. Just like the example in #61250 shows.

If we are using `sets.String`, we will ignore the `toleration` with the same key.

Reply:
What I was suggesting, was to use the `sets.String` to sort the keys like before and get rid of duplicates. But looking once again at your solution I think your approach is much better, thanks!